requirements.txt

```diff
@@ -2,6 +2,7 @@ flask
 flask_sqlalchemy
 flask_marshmallow
 flask_migrate
+Flask-RESTful==0.3.10
```

Comment:
Remove the version information to avoid errors.

Reply:
Ack